packages/client/src/client.ts

```diff
@@ -74,6 +85,7 @@ class Client<FetcherRequestInit extends RequestInit = RequestInit> {
     fetchOptions?: FetcherRequestInit;
     channelId?: string;
     errorPolicy?: 'none' | 'all' | 'ignore';
+    validateCustomerAccessToken?: boolean;
```

Comment:
Is there a reason why this is optional?

Reply:
@jorgemoya Yep, it's optional so that we don't have to update every single `client.fetch` in Catalyst to get this behavior. It's optional, but defaults to `true`, so if we want to override the behavior somewhere, we can.

Reply:
Yeah I guess my question was more in the sense of why would we want to override it but having the optional override shouldn't hurt.